php/Terminus/Models/MachineToken.php

```diff
@@ -17,15 +19,17 @@ public function __construct($attributes, array $options = []) {
 
   /**
    * Deletes machine token
-   *
-   * @return array
+   * @return null
```

Comment:
@greg-1-anderson This repo is already divergent.
@ronan Since PHP always returns null in the absence of a return, the difference between `@return void` and `@return null` is that `void` denotes to users that its return value will be of no use to them. `null` is used in conjunction with other possible return types, e.g. `@return null|string`